test/e2e/util.go

```diff
@@ -2193,10 +2193,28 @@ func restartKubeProxy(host string) error {
 	if !providerIs("gce", "gke", "aws") {
 		return fmt.Errorf("unsupported provider: %s", testContext.Provider)
 	}
-	_, _, code, err := SSH("sudo /etc/init.d/kube-proxy restart", host, testContext.Provider)
+	_, _, code, err := SSH("sudo /bin/sh -c 'pgrep kube-proxy | xargs kill'", host, testContext.Provider)
```

Comment:
not `pkill`? also what is restarting the proxy, now that we're not calling restart ourselves? a comment would probably be helpful